brainTeasers/waterJugs.md

```diff
@@ -1,3 +1,5 @@
 You have a five-quart jug, a three-quart jug, and an unlimited supply of water (but no measuring cups).
 How would you come up with exactly four quarts of water?
 Note that the jugs are oddly shaped, such that filling up exactly "half" of the jug would be impossible.
+
+measure the 5, pour into the 3 to get 2 quarts remaining in the 5 quart. Do this twice.
```

Comment:
How could you do this twice if there are 2 quarts still in the 5 quart jug?. You'd need a place to store the water you're measuring..